### ISSUE TITLE
Latest version generates errors during build

### ISSUE THREAD

**Issue by tai-flywire** [deckgo/gatsby-remark-highlight-code#10]
Using the latest version of gatsby-remark-highlight-code(1.3.2) and @deckdeckgo/highlight-code(1.0.4), I get the following error during `gatsby build`:

```
(node:25265) UnhandledPromiseRejectionWarning: TypeError: head.querySelector is not a function
    at bootstrapLazy (/Users/tai.nguyen/Documents/GitHub/commercial-documentation/public/render-page.js:2473:42)
    at /Users/tai.nguyen/Documents/GitHub/commercial-documentation/public/render-page.js:2800:72
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

The stack trace isn't very helpful but removing usage of the component gets rid of the error.

**Comment by peterpeterparker** [comment]
@tai-flywire thank you for reporting the issue. it seems liked to the Stencil compiler, therefore I opened the following issue https://github.com/ionic-team/stencil/issues/2411

I actually face the problem with the component too but I'm still able to compile as my build is ignoring the error. I think  it's because I have not yet upgraded `react-helmet` as  you can notice in the dependencies of my website https://github.com/peterpeterparker/daviddalbusco.com.

**Comment by peterpeterparker** [comment]
The issue in Stencil has been resolved and I tested the pre-version v1.13.0-3 which does solve the issue too. I'll integrate it as soon as officially released in the highlight-code component and it will solve the issue.

**Comment by peterpeterparker** [comment]
I've just released `@deckdeckgo/highlight-code` v1.0.5 and `gatsby-remark-highlight-code` v1.3.3 which solve this issue.

Thank you for reporting the problem @tai-flywire 👍
